x/launch/types/message_trigger_launch.go

```diff
@@ -9,7 +9,7 @@ const TypeMsgTriggerLaunch = "trigger_launch"
 
 var _ sdk.Msg = &MsgTriggerLaunch{}
 
-func NewMsgTriggerLaunch(coordinator string, launchID, remainingTime uint64) *MsgTriggerLaunch {
+func NewMsgTriggerLaunch(coordinator string, launchID uint64, remainingTime int64) *MsgTriggerLaunch {
```

Comment:
We should add the check `remainingTime>0` in `ValidateBasic` with a test

Reply:
true! thanks for the suggestion

Reply:
true! thanks for the suggestion

Reply:
I've added a test case for `remainingTime<=0` only in the test for `ValidateBasic` following the testing conventions we have discussed 👍